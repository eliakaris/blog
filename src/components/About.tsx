import * as React from 'react';

import './About.css';

function About() {
  return (
    <div>
      <img className="me" src="/img/me3.jpg" />
      <h2>About me</h2>
      <p>Hey there. My name is Elia Karagiannis and I live in Seattle, Washington.</p>

      <h2>Professional</h2>
      <p>I'm a software developer at Pinterest working on advertiser partner experiences.
        I've previously worked at Microsoft for 10 years until 2017.
        Before that I worked in the bay area as a developer and before that in Springfield, MO.
      </p>

      <h2>Hobbies</h2>
      <p><a href="http://github.com/eliakaris">programming</a>, 
      <a href="http://www.bing.com/search?q=rams+football">sports</a></p>
    </div>
  );
}

export default About;