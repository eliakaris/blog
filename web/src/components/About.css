.me {
  float: right;
  margin-left: 20px;
  border-radius: 4px;
  box-shadow: 5px 5px 5px #ccc;
}
