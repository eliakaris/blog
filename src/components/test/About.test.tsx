import * as React from 'react';
import About from '../About';
import * as ShallowRenderer from 'react-test-renderer/shallow';
import * as ReactSixteenAdapter from 'enzyme-adapter-react-16';
import { configure } from 'enzyme';

configure({ adapter: new ReactSixteenAdapter() });

describe('<About /> tests', () => {
  let renderer: any;

  beforeAll(() => {
    renderer = ShallowRenderer.createRenderer();
  });

  it('renders a snapshot', () => {
    const component = renderer.render(<About />);
    expect(component).toMatchSnapshot();
  });
});