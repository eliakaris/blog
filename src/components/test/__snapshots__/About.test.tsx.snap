// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> tests renders a snapshot 1`] = `
<div>
  <img
    className="me"
    src="/img/me.png"
  />
  <h2>
    About me
  </h2>
  <p>
    Hi there. My name is Elia Karagiannis and I live in Seattle, Washington.
  </p>
  <h2>
    Professional
  </h2>
  <p>
    I'm a software development manager at Pinterest Seattle.  My team works on content creation for web and mobile. I previously worked at Microsoft for 10 years until 2017. Before that I worked in the bay area as a developer and before that in Springfield, MO.
  </p>
  <h2>
    Hobbies
  </h2>
  <p>
    <a
      href="http://github.com/eliakaris"
    >
      programming
    </a>
    ,
    <a
      href="http://www.bing.com/search?q=rams+football"
    >
      sports
    </a>
  </p>
</div>
`;
